relayer/pkg/chainlink/ocr2/utils.go

```diff
@@ -96,6 +99,7 @@ func isEventFromContract(event *caigotypes.Event, address string, eventName stri
 	return event.Keys[0].Cmp(eventKey) == 0
 }
 
```

Comment:
Need to re-enable the address check in the method above (isEventFromContract)

Reply:
For address sanitization, I like the `HexToBytes` function we have in the ocr2keys test and we probably want a few helpers for the reverse: `"0x"+hex.EncodeToString(starknet.PadBytes(felt.Bytes(), 32))` etc